src/mame/drivers/at.cpp

```diff
@@ -27,7 +25,6 @@ RAM: 1MB - 8MB on board
 Mass storage: 1.44MB Floppy disk drive and 80MB IDE hard disk
 On board: 2xserial, parallel, bus mouse, keyboard
 To-Do: Complains about missing mouse hardware (Bus Mouse), hangs in POST
-
```

Comment:
This is yet another blank line before a heading that aided readability that you’ve removed.